### DISCUSSION TITLE
Have default setting for Specials

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#10741] [Ideas & Feature requests]
New specials come out and I keep missing them because they get loaded as skipped.
I would like all new specials to be set to wanted..
Also when I add a new show I would like the specials also be set to wanted..
Also I would want them even if they haven't aired.

**Comment by p0psicles** [comment]
We could add an option for that. But do realize that searching for specials is mostly a hit and miss. Cause the specials often don't follow a certain format

**Reply by nydave69** [reply]
> We could add an option for that. But do realize that searching for specials is mostly a hit and miss. Cause the specials often don't follow a certain format

Yea I know, but I seem to have been really lucky so far...
